config/webpack/webpack.config.js

```diff
@@ -291,6 +291,9 @@ const makeWebpackConfig = ({
         library: 'static',
         libraryTarget: 'umd2',
       },
+      optimization: {
+        nodeEnv: process.env.NODE_ENV,
```

Comment:
This ensures the `NODE_ENV` is set correctly. Otherwise it defaults to the webpack mode which is always `development` for the render config.